tests/integration/handling-api-messages.js

```diff
@@ -0,0 +1,20 @@
+describe("handling api messages", function () {
+
+  it("handles and re-hydrates commands", function () {
```

Comment:
yeah this was a bad description – what i mean is that the command is serialized and deserialized automatically. That's why the test app can just send it without any modifications